docker-cloud/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# syntax=docker/dockerfile:1
+
+## Build
+FROM golang:1.19-bullseye as build
+
+WORKDIR /app
+
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
+
+COPY *.go ./
+
+RUN go build -o /out
+
+CMD [ "/out" ]
```

Comment:
Does this `CMD` do anything if it's just in a throwaway container for building?

Reply:
It does not, thanks